src/interface/wallet.h

```diff
@@ -251,6 +288,38 @@ struct WalletBalances
     }
 };
 
+// Wallet transaction information.
+struct WalletTx
+{
+    CTransactionRef tx;
+    std::vector<isminetype> txin_is_mine;
+    std::vector<isminetype> txout_is_mine;
+    std::vector<CTxDestination> txout_address;
+    std::vector<isminetype> txout_address_is_mine;
```

Comment:
This may not be practical to change for legacy reasons, but I find the naming here confusing; it makes me think bool and not vector. Something like `my_txout_addresses` (and likewise for two others above) seems clearer.

Reply:
Oh, you're totally right; I misunderstood the content of these vectors. Thanks for the clarification.